pydarn/logging_config.yaml

```diff
@@ -29,9 +29,9 @@ handlers:
         formatter: pydarn_fmt
```

Comment:
Why is the time format "2018-11-05 15:39:49,678" instead of using a decimal (example: 2018-11-05 15:39:49.678)? It doesn't matter that much I guess, but standard US/UK english uses decimals.

Reply:
Apparently, that comma is built in with the python's `asctime` function. I can look at trying to change it but if it is not a big deal then I won't stress about too much right now. 

Personally, I thought it was a typo on my part :p